app/views/documents/index.html.erb

```diff
@@ -131,10 +131,15 @@
             <p class="govuk-body govuk-!-margin-bottom-1">
               <%= link_to image_tag(document.file.representation(resize: "300x212")),
                           url_for_document(document), target: :_blank %>
+            </p>
             <p class="govuk-body">
-              <%= document.name %><br/>
-              <%= link_to "View in new window", url_for_document(document), target: :_new %>
+              <%= document.name %></br>
             </p>
+            <%= link_to "View in new window", url_for_document(document), target: :_new %>
+            <p class="govuk-body">
+              <%= form_for document, url: planning_application_document_unarchive_path(document_id: document.id), class: "restore", method: :patch do |form| %>
```

Comment:
I don't think you need to specify `method: :patch` here, as `document` is an existing record, so Rails will do magic. Not important though.

Reply:
Ah, didn't realise. Will remove.